geopandas/array.py

```diff
@@ -1044,6 +1044,34 @@ def isna(self):
         else:
             return np.array([g is None for g in self.data], dtype="bool")
 
+    def value_counts(
+        self,
+        dropna: bool = True,
+    ):
+        """
+        Compute a histogram of the counts of non-null values.
+
+        Parameters
+        ----------
+        dropna : bool, default True
+            Don't include counts of NaN
+
+        Returns
+        -------
+        pd.Series
+        """
+
+        # note ExtensionArray usage of value_counts only specifies dropna,
+        # so sort, normalize and bins are not arguments
+        values = to_wkb(self)
+        from pandas import Series, Index
+
+        result = Series(values).value_counts(dropna=dropna)
+        # value_counts converts None to nan, need to convert back for from_wkb to work
+        # note result.index already has object dtype, not geometry
+        result.index = Index(from_wkb(result.index.fillna(None)))
```

Comment:
It seems that this `fillna(None)` doesn't work for all (recent?) pandas versions, see the failing tests (didn't yet check in detail)

Reply:
I'm not exactly sure what happened there, in the process of making a new local env to try and replicate